tests/storage/upgrade/conftest.py

```
@@ -24,54 +24,103 @@
+def override_cdiconfig_scratch_spec(
+    hyperconverged_resource_scope_session,
+    cdi_config,
+    storage_class_for_updating_cdiconfig_scratch,
+):
+    """
+    Change spec.scratchSpaceStorageClass to the selected StorageClass on CDIConfig.
+    """
+    if storage_class_for_updating_cdiconfig_scratch:
+        new_sc = storage_class_for_updating_cdiconfig_scratch.name
+
+    with update_scratch_space_sc(
```

Ruff (0.15.14)

[error] Undefined name `update_scratch_space_sc`

(F821)